engine/userguide/storagedriver/btrfs-driver.md

```diff
@@ -65,8 +65,8 @@ This procedure is essentially identical on SLES and Ubuntu.
     the contents of `/var/lib/docker/`:
 
     ```bash
-    $ sudo cp -au /var/lib/docker /var/lib/docker.bk
-    $ sudo rm -rf /var/lib/docker/*
+    $ sudo mv /var/lib/docker /var/lib/docker.bk
+    $ sudo mkdir  /var/lib/docker/
```

Comment:
The rights depend on the filesystem of the mounted share and not on the mount point. I've done it this way without any problems.